wslpy/system.py

```diff
@@ -30,13 +30,26 @@ def __shellEnvVar__(regname):
         print(err)
 
 
+def _regEnvVarList_():
```

Comment:
What you are doing here is getting the System Environment Varibale; What we want to archive here is to get the user variables from the registry such as users Desktop folder, Documents folder, etc, from registry `HKCU:\Software\Microsoft\Windows\CurrentVersion\Explorer\User Shell Folders`.